### PR TITLE
Adding grids to tables

### PR DESCRIPTION
**Pull request by adamoutler**
The grid makes it so the tables don't look like columns of data.  I feel this increases readability. 
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by adamoutler** [comment]
Picture of old table and new
image
image

**Comment by adamoutler** [comment]
Please note, I am not, nor do I consider myself an artist.  Please feel free to suggest or edit in other colors.  I just want grid lines.

File: source/stylesheets/prism.css [hunk 1]
```diff
@@ -371,3 +371,7 @@ div.code-toolbar > .toolbar span:focus {
 	content: attr(data-prompt);
 }
 
+table, th, td {
+   border: 1px solid #d5d2ff;
+   background: #dfe6f0;
+}
```
Comment (frenck) [inline source/stylesheets/prism.css]: This is the stylesheet for the prism code highlighter. Please use the available scss files to adjust other styling.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/stylesheets/prism.css (hunk 1 above)
Comment (adamoutler) [reply source/stylesheets/prism.css]: Moved to layout.css

**Comment by frenck** [comment]
Discussed it with the frontend guys and we are going to friendly decline this change. For the future, we might look into zebra-style rows, and bringing them closer to the material style we have in Home Assistant.

**Pull request closed without merging**